### ISSUE TITLE
Allow for "appearance" value "radio"

### ISSUE THREAD

**Issue by dlockhart** [medikoo/csslint-next#1]
The [official draft spec](https://drafts.csswg.org/css-ui-4/#appearance-switching) seems to be heading in the direction of only allowing `auto` and `none` as possible values for the `appearance` property. However, [at least Firefox](https://developer.mozilla.org/en-US/docs/Web/CSS/-moz-appearance) supports a value of `radio`, and [this article](https://css-tricks.com/almanac/properties/a/appearance/) implies that Webkit does as well.

Currently using `appearance: radio;` causes csslint-next to output the following warning:

```
Expected (icon | window | desktop | workspace | document | tooltip | 
dialog | button | push-button | hyperlink | radio-button | checkbox | 
menu-item | tab | menu | menubar | pull-down-menu | pop-up-menu | 
list-menu | radio-group | checkbox-group | outline-tree | range | field | 
combo-box | signature | password | normal | none | inherit) but found 'radio'.
```

The code causing this is on [line 3644 of parserlib.js](https://github.com/medikoo/csslint-next/blob/master/lib/parserlib.js#L3644).

**Comment by medikoo** [comment]
@dlockhart Thanks for this request. Still it'll be way better if it comes directly to original [parserlib](https://github.com/CSSLint/parser-lib/), and through that ported here. Have you tried to request that change to original repository? 

This project just adds some modifications to original parserlib, and should derive as minimal as possible from original.

**Comment by medikoo** [comment]
I'm closing it for now. I'll be happy to take it with parserlib core update

**Comment by dlockhart** [comment]
Gotcha -- I'll take that route for now, thanks.

**Comment by dlockhart** [comment]
Submitted a [pull request to parserlib](https://github.com/CSSLint/parser-lib/pull/171) to resolve this. Will update if it gets merged in.

**Comment by medikoo** [comment]
Ok, I look forward

**Comment by dlockhart** [comment]
hey @medikoo, the [PR into parserlib](https://github.com/CSSLint/parser-lib/pull/171) that fixes this was just merged.

**Comment by medikoo** [comment]
Upgraded internal parserlib, published as v1.1.0. Thank you!
